src/prefs/MenuEditorPage.js

```diff
@@ -978,7 +978,7 @@ var MenuEditorPage = class MenuEditorPage {
           .replace('%s', 'https://github.com/Schneegans/Fly-Pie/issues'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('Deep hierarchies are fairly inefficient. Avoid putting menus into menus in menus.'),
+      _('Keep menu hierarchies efficient by putting the right amount of menus into new ones.'),
```

Comment:
The point is, that you should prefer depth over breadth when designing menus. For maximum efficiency a menu level should contain at most 8 items. Twelve is OK as well, but eight is better. If you need more, then go deeper!

You could aleays create additional menus, but this is not always desirable as this requires learning more shortcuts.

Maybe something like this?

```suggestion
      _(' Prefer depth over breadth when designing menus. For maximum efficiency, create nested menus when you need more then eight items.'),
```

Reply:
It is neither obvious that a menu gets broader, nor  what "nesting" actually is.
You either get the point beforehand, or are none the wiser with the explanation.

Reply:
In my opinion this should be easy to understand. However, I still like the original string "Deep hierarchies are pretty efficient. Put menus into menus in menus!" most. Maybe we should just leave it like this :wink: